cert-data.yml

```diff
@@ -1,4 +1,179 @@
 ---
+### 4_programmatic Default course
+Default/OpenEdX/Certificate: &cert_defaults
+  VERSION: 4_programmatic
+  TEMPLATEFILE: False
+  instructors:
+    - instructor: &instructor_1
+        x_position: 475
```

Comment:
We should document each of these with something intuitive like "top right" or "far right, 3rd from top".